src/ESLintLinter.php

```diff
@@ -0,0 +1,128 @@
+<?php
```

Comment:
Mind adding our license comment header to this file? I should have mentioned that earlier. You can just copy it from one of the other files in this directory.

Reply:
done